grobidTagger.py

```diff
@@ -239,6 +247,12 @@ class Tasks:
     parser.add_argument("--use-ELMo", action="store_true", default=False, help="Use ELMo contextual embeddings.")
     parser.add_argument("--output", help="Directory where to save a trained model.")
     parser.add_argument("--input", help="Grobid data file to be used for training (train action), for trainng and evaluation (train_eval action) or just for evaluation (eval action).")
+    parser.add_argument("--ignore-features", default=False, action="store_true", help="Ignore layout features")
+    parser.add_argument(
+        "--feature-indices",
+        type=parse_number_ranges,
+        help="The feature indices to use. e.g. 7:10. If blank, all of the features will be used."
```

Comment:
minor: currently the syntax is `7-10` (i.e. more like when selecting pages to print than Python or mathlab syntax)